commons/src/main/java/com/flipkart/flux/commons/dto/Eventsinfo.java

```diff
@@ -0,0 +1,31 @@
+package com.flipkart.flux.commons.dto;
```

Comment:
These are api interfaces right, why are we keeping them in commons module? Can we have an interface module instead. We can share the com.flipkart.flux:interface:$version jar to the client to use. To me commons module is something internal to the core/runtime engine being built.